build/Common.props

```diff
@@ -54,6 +54,10 @@
     <PackageReference Include="Microsoft.CodeCoverage" Version="$(MicrosoftCodeCoveragePkgVer)" Condition="'$(Configuration)'=='Release'" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IncludeSharedExceptionExtensionSource)'=='true'">
```

Comment:
```suggestion
  <ItemGroup Condition="'$(IncludeSharedExceptionExtensionsSource)'=='true'">
```

Reply:
Solved in 6cb3f09044d73cee3f3a3415204f1900497c80e4